### PR TITLE
Sequelize setup

### PR DESCRIPTION
**Pull request by ladystephani**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -12,6 +12,23 @@ app.get("/", (req, res) => {
   res.send("hi");
 });
 
+// Helper functions:
+function searchProducts() {
```
Comment (dorvilsteven) [inline server.js]: comment this line out

File: server.js [hunk 2]
```diff
@@ -12,6 +12,23 @@ app.get("/", (req, res) => {
...
+  if (query.brandName) {
```
Comment (dorvilsteven) [inline server.js]: comment this line too

**Review by dorvilsteven — CHANGES_REQUESTED** [review]
just comment those lines in server.js

**Comment by ladystephani** [comment]
Commented out

**Review by dorvilsteven — APPROVED** [review]

**Pull request merged**